static/styles/alerts.scss

```diff
@@ -66,17 +175,15 @@
             font-family: "FontAwesome";
             font-size: 1.3em;
             content: "\f071";
-
-            color: hsl(16, 60%, 55%);
         }
 
         &.alert-error {
-            color: hsl(0, 80%, 40%);
-            border: 1px solid hsl(0, 80%, 40%);
-            box-shadow: 0 0 2px hsl(0, 80%, 40%);
+            color: $alert-error-red;
```

Comment:
Similarly, you should extract a prep commit for extracting these colors to variables.